src/main/java/Duke.java

```diff
@@ -1,10 +1,49 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Ui ui = new Ui();
+        Storage storage = new Storage("\\save.txt");
+        ui.printWelcome();
+        Scanner sc = new Scanner(System.in);
+        while(sc.hasNextLine()) {
+            String next = sc.nextLine();
+            if(next.equals("bye") ) {
```

Comment:
Perhaps you could add a whitespace after "if" since it is a Java reserved word? I noticed a similar issue in other areas well.